server/pulp/plugins/util/publish_step.py

```diff
@@ -45,6 +45,20 @@ def _post_order(step):
         yield v
     yield step
 
+def _safe_makedirs(*args):
```

Comment:
If you're going to do `(*args)` instead of matching the signature in `os.makedirs()`, it would be better to do `(*args, **kwargs)` in case the user calls as `_safe_makedirs(path, mode=0775)`.